mgmt_cli_test.py

```diff
@@ -44,7 +44,7 @@ def test_mgmt_repair_nemesis(self):
             1) Run cassandra stress on cluster.
             2) Add cluster to Manager and run full repair via Nemesis
         """
-        self._generate_load()
+        self.generate_load_and_wait_for_results()
```

Comment:
since generate_load_and_wait_for_results calls _generate_load and waits for results, the following lines of _generate_load() are not needed and may be deleted if they have no other use cases:
```
        self.log.info('Sleeping for 15s to let cassandra-stress run...')
        time.sleep(15)
```

Reply:
this sleep doesn't make a difference, if we are anyways waiting for the results, no?